query/reverse.js

```diff
@@ -39,7 +40,7 @@ function generateQuery( clean ){
 
   // layers
   if( _.isArray(clean.layers) && !_.isEmpty(clean.layers) ) {
-    vs.var( 'layers', clean.layers);
+    vs.var( 'layers', _.difference(clean.layers, peliasConfig.api.targets.layer_aliases.coarse));
```

Comment:
Sorry I didn't mention this before, but there is a `get()` method on the config which handles the case where this path doesn't exist for some reason, which would otherwise result in an error.

ie. if `api.targets` didn't exist for some reason.

If I recall correctly, it would look like this, the second param being a default value if the path is not resolvable:
```
peliasConfig.get('api.targets.layer_aliases.coarse', [])
```

ref: https://github.com/pelias/config/blob/master/index.js#L80

Reply:
Fixed 👍